libs/api-mocks/msw/util.ts

```diff
@@ -69,11 +69,6 @@ export const paginated = <P extends PaginateOptions, I extends { id: string }>(
 export const repeat = <T extends { id: string; name: string }>(obj: T, n: number): T[] =>
   new Array(n).fill(0).map((_, i) => ({ ...obj, id: obj.id + i, name: obj.name + i }))
 
-export const clone = <T extends object>(obj: T): T =>
-  typeof structuredClone !== 'undefined'
-    ? structuredClone(obj)
-    : JSON.parse(JSON.stringify(obj))
-
```

Comment:
This was kind of interesting — my switch to `Map` for `db.diskBulkImportState` would not work with this `JSON.parse(JSON.stringify(obj))` fallback. Fortunately I don't think there is any situation where this code would hit the fallback. Two-year-old browsers only, basically.

https://caniuse.com/mdn-api_structuredclone